parser/inline.go

```diff
@@ -1201,7 +1201,7 @@ func helperEmphasis(p *Parser, data []byte, c byte) (int, ast.Node) {
 		}
 
 		if i+1 < len(data) && data[i+1] == c {
-			i++
+			i += 2
```

Comment:
`i++` guarantees the next character checked is `c`.  This means `helperFindEmphChar` above returns a guaranteed `0` for length, which then short-circuits to the `return 0, nil` case.  We choose to skip the next `c` character here by incrementing by 2 instead.  (This will expose a separate bug if we're actually in the triple `c` case)